python/tidy/servo_tidy/tidy.py

```diff
@@ -101,8 +101,8 @@ def filter_file(file_name):
     return True
 
 
-def filter_files(start_dir, faster, progress):
-    file_iter = get_file_list(start_dir, faster, ignored_dirs)
+def filter_files(start_dir, only_changed_files, progress):
```

Comment:
I changed this file a little bit, because I think that `only_changed_files` is a little more descriptive than `faster`